src/material/core/focus-indicator/_focus-indicator.scss

```diff
@@ -0,0 +1,95 @@
+@import '../theming/theming';
+
+/// Mixin that turns on strong focus indicators.
+///
+/// @example
+///   .my-app {
+///     @include mat-strong-focus-indicators();
+///   }
+
```

Comment:
I think you would want to omit this blank line so that the doc affects the mixin (same as JsDoc)

(same below)